relion/tests/test_projection_subtraction_no_relion.py

```diff
@@ -0,0 +1,165 @@
+# ***************************************************************************
+# * Authors:    Marta Martinez (mmmtnez@cnb.csic.es)
+# *             Roberto Marabini (roberto@cnb.csic.es)
+# *
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
```

Comment:
please use GPL3

Reply:
I understand than the suggestion is to change version 2 by version 3. done